### PR TITLE
DRILL-6017 Fix for SHUTDOWN button being visible for non Admin users

### PR DESCRIPTION
**Pull request by bitblender**
This fix repurposes an earlier change I had made to show admin user info, to selectively enable the SHUTDOWN button.

### CODE REVIEW

**Comment by arina-ielchiieva** [comment]
+1, LGTM.

File: exec/java-exec/src/main/resources/rest/index.ftl [hunk 1]
```diff
@@ -112,7 +114,7 @@
       </div>
   </div>
 
-   <#if model.shouldShowUserInfo()>
+   <#if model.shouldShowAdminInfo()>
```
[the remark this answers is not in the document]
Comment (bitblender) [reply exec/java-exec/src/main/resources/rest/index.ftl]: I have added the check around the shutdown JS code.

File: exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java [hunk 2]
```diff
@@ -189,7 +189,7 @@ public ClusterInfo getClusterInfoJSON() {
     String adminUsers = ExecConstants.ADMIN_USERS_VALIDATOR.getAdminUsers(optionManager);
     String adminUserGroups = ExecConstants.ADMIN_USER_GROUPS_VALIDATOR.getAdminUserGroups(optionManager);
 
-    final boolean shouldShowUserInfo = isUserLoggedIn &&
+    final boolean shouldShowAdminInfo = isUserLoggedIn &&
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java]: I think it is not fully correct to use this option to decide if we can show shutdown button or not.
This option is used only when authentication is enabled. When it's not, user by default is admin. So with your fix without authentication, this button is also absent.

**Comment by arina-ielchiieva** [comment]
+1 but I have created Jira to display shutdown button when authentication is not enabled since in that case user is admin by default - https://issues.apache.org/jira/browse/DRILL-6021.

**Pull request closed without merging**